conf/messages.conf

```diff
@@ -289,7 +289,8 @@
 287: You cannot change party leaders in this map.
 //Missing stuff for @killer related commands.
 288: You are no longer killable.
-//289-290 FREE
+289: Target character must be in the same map.
+//290 FREE
```

Comment:
Hmm, Not working on 2010 ~ 2011 clients. NO MSG is showing. I'll check later what is the exact date of the client supports this feature.